examples/jobs/main.go

```diff
@@ -49,13 +49,16 @@ func main() {
 		panic(err)
 	}
 
+	maxRetries := uint32(4)
+	interval := time.Second * 3
 	job := daprc.Job{
 		Name:     "prod-db-backup",
 		Schedule: "@every 1s",
 		Repeats:  10,
 		Data: &anypb.Any{
 			Value: jobData,
 		},
+		FailurePolicy: daprc.NewFailurePolicyConstant(&maxRetries, &interval),
 	}
```

Comment:
I think from this example it's clear that using the addresses of an int or duration adds unnecessary faff in my opinion